code/game/say.dm

```diff
@@ -35,8 +35,32 @@ GLOBAL_LIST_INIT(freqtospan, list(
 /atom/movable/proc/Hear(message, atom/movable/speaker, message_language, raw_message, radio_freq, list/spans, list/message_mods = list())
 	SEND_SIGNAL(src, COMSIG_MOVABLE_HEAR, args)
 
-/atom/movable/proc/can_speak()
-	//SHOULD_BE_PURE(TRUE)
+
+/**
+ * Attempts to / checks if our movable can speak the passed message.
+ * CAN include feedback messages about why someone can or can't speak
+ *
+ * Used in [proc/say] and other methods of speech (radios) after a movable has inputted some message.
+ * If you just want to check if the movable is able to speak in character, use [proc/can_speak] instead.
+ */
+/atom/movable/proc/try_speak(message, ignore_spam = FALSE, forced = FALSE)
```

Comment:
Why does 'can' have feedback messages but 'try' doesnt, try implies you are doing, can only implies it's possible.

Reply:
i dont know if im stupid or im being gaslit